swagger.yaml

```diff
@@ -155,5 +317,116 @@ components:
           format: date
         assignedPersons:
           type: array
-          items: 
-            type: integer
\ No newline at end of file
+          items:
+            type: integer
+
+    Leads:
+      type: object
+      description: Lead data from Pipedrive API
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
```

Comment:
What does this mean?

Reply:
Same case as the pipedrive-interest-key-here. I commented more about this to the Interest schema at the bottom.